index.html

```diff
@@ -44,6 +44,9 @@
   </header>
     <div class="ui-widget">
     <nav>
+      <div class="horizontal-lmht"></div>
+      <a href="https://github.com/LMHT/LMHT.github.io/blob/master/CONTRIBUTING.md" target="_blank" class="horizontal-ribbon"><img width="40" src="img/github.svg"><div>Contribute</div>
```

Comment:
Consider making the image and div tags on their own line.  This one is pretty long.